crates/cli-support/src/js/mod.rs

```diff
@@ -1971,16 +1984,59 @@ impl<'a> Context<'a> {
             .types
             .get::<ast::WebidlFunction>(binding.webidl_ty)
             .unwrap();
-        let mut builder = binding::Builder::new(self);
-        builder.catch(catch)?;
-        let js = builder.process(&binding, &webidl, false, &None, &mut |cx, prelude, args| {
-            cx.invoke_import(&binding, import, bindings, args, variadic, prelude)
-        })?;
-        let js = format!("function{}", js);
+        let js = match import {
+            AuxImport::Value(AuxValue::Bare(js))
```

Comment:
I think that this may not be quite right in terms of matching where *technically* any value imported is a candidate for not requiring any glue. For example the other values, getters/setters, can all be hooked up directly in theory. They today, however, all require a `method` invocation style and will fail the `Static` check below very quickly, so it effectively ends up being the same. 

Perhaps though it'd be good to refactor this in a forward-facing way where we match `AuxImport::Value(v)` here and then have shared code between way below in here about how to turn `v` into a JS snippet?